### ISSUE TITLE
header items

### ISSUE THREAD

**Issue by DanVanAtta** [williamdekryger/triplea-game.github.io#4]
One:

I recommend first header item being: "Download"

Top left corner, we want people to download the game! They may arrive from other pages than just the front page, so having the link to download available from everywhere is good. It also sets the stage for the order of operations we want people to go in, namely to then next click the "learn to play" link while we wait for them to download.

Two:
Replace "Documentation" with "Learn to Play" (?)

Three:
Remove "Contribute" and place in footer only

Four:
As discussed in #1 , remove "news"

Five:
Maps and Community are perfect. Maps shows off more of the features of the game, obvious eye candy. Community is a nice last header item to indicate that once you progress more, there is a lot of community interaction to be had.

So to sum up, I recommend the header contain these items:

- logo image (links to homepage, exists today but is not a link)
- triplea (links to homepage, exists today)
- download (DNE)
- learn to player (exist today but is called "Documentation")
- maps (exists today)
- community  (exists today)

**Comment by williamdekryger** [comment]
At one point I had a download link there, but I was told that it was better to remove it: http://tripleadev.1671093.n2.nabble.com/TripleA-Website-Redesign-Migration-to-Github-Pages-tp7589306p7589331.html

**Comment by DanVanAtta** [comment]
I didn't see the discussion on why it would be better to remove it. The DL link in top left is relatively important - key to ensuring that everyone will notice the DL link right away, and no matter which page is their entry point to the website.

Having our header links reflect this desired new path of users I think is very wise:
- download
- learn to play
- maps/features
- community

Not only do we get reasonable headers to organize content, but we also have this great second benefit where we communicate by design to a new player exactly what they should do if they want to become engaged with the game. If we skip the DL link, we'll need to have that information in the learn to play section for certain (and this begins introducing duplication, both a maintenance cost and also a smell we have design problems). Likely we wouldn't include the DL link on every page, in that case the site navigation is handicapped IMO.

**Comment by williamdekryger** [comment]
Yeah, I could hide the download link on the homepage to avoid duplication, but keep it on the subpages. We'll discuss the other ones.

**Comment by williamdekryger** [comment]
Addressed in 85d301ffd1548df262105852752cddd0ff7e04b5
